app/controllers/online_applications_controller.rb

```diff
@@ -4,6 +4,7 @@ class OnlineApplicationsController < ApplicationController
   def edit
     authorize online_application
     @form = Forms::OnlineApplication.new(online_application)
+    @form.update_attributes(jurisdiction_id: current_user.jurisdiction_id)
```

Comment:
I would personally try to wrap this functionality somewhere else, similarly like we have the `ApplicationBuilder` for `Application` (even though of course that's an actual builder and more complex). It looks like logic the controller should not worry about. 

Perhaps we could have `assign_default_jurisdiction` on the `OnlineApplication` which accepts `current_user` and does it there. That would simplify testing that particular piece (cause it would be in the model). And in the controller test bellow, you can just check that method has been called on the model.